class-amp-img.php

```diff
@@ -5,6 +5,11 @@
 class AMP_Img_Converter extends AMP_Converter {
 	public static $tag = 'img';
 
+	private $anim_extensions = '/.(gif|gifv)/';
```

Comment:
Yeah, I wasn't really sure, I actually asked in [SO](https://stackoverflow.com/questions/34501955/what-are-the-supported-formats-for-amp-anim) about what file formats should `<amp-anim>` be used with, but to no avail so far. I can remove it for `.gif` only and we can add more formats if necessary later.